### DISCUSSION TITLE
Statement Timeout Error

### DISCUSSION THREAD

**Discussion by spyriaev** [supabase/supabase#21164] [Questions] [answered]
While attempting to alter a table within our database, we are facing a persistent error.

The  operation we are trying to perform is simple the addition of a new column to our table. The exact statement being executed is as follows:

```sql
ALTER TABLE "stats"."look_views" ADD COLUMN "user_agent" JSON;
```

Unfortunately, every time we attempt to run this statement, we are met with the following error:

`canceling statement due to statement timeout`

The table  `"stats"."look_views"` has 2 million rows and size 158 Mb.

**Comment by GaryAustin1** [comment, marked as answer]
See this: https://github.com/orgs/supabase/discussions/21133
